### PR TITLE
CAS-372 implement loading state

### PR DESCRIPTION
**Pull request by bychkovdmitry**
https://stream-io.atlassian.net/browse/CAS-372

### Description

Implemented loading state with shimmer effect

### Checklist

- [X] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [X] PR targets the `develop` branch
- [ ] Changelog updated with client-facing changes
- [ ] New code is covered by unit tests
- [X] Comparison screenshots added for visual changes
- [X] Reviewers added

### CODE REVIEW

File: buildSrc/src/main/kotlin/com/getstream/sdk/chat/Dependencies.kt [hunk 1]
```diff
@@ -69,6 +69,7 @@ private const val TRUTH_VERSION = "1.1"
 private const val WORK_VERSION = "2.4.0"
 private const val FLIPPER_VERSION = "0.63.0"
 private const val FLIPPER_SO_LOADER_VERSION= "0.9.0"
+private const val SHIMMER_VERSION= "0.5.0"
```
Comment (zsmb13) [inline buildSrc/src/main/kotlin/com/getstream/sdk/chat/Dependencies.kt:72]: Please add this alphabetically to the list (and move the Flipper version, if we're already making changes here)

File: buildSrc/src/main/kotlin/com/getstream/sdk/chat/Dependencies.kt [hunk 2]
```diff
@@ -171,6 +172,7 @@ object Dependencies {
     const val flipper = "com.facebook.flipper:flipper:$FLIPPER_VERSION"
     const val flipperNetwork = "com.facebook.flipper:flipper-network-plugin:$FLIPPER_VERSION"
     const val flipperLoader = "com.facebook.soloader:soloader:$FLIPPER_SO_LOADER_VERSION"
+    const val shimmer = "com.facebook.shimmer:shimmer:$SHIMMER_VERSION"
```
Comment (zsmb13) [inline buildSrc/src/main/kotlin/com/getstream/sdk/chat/Dependencies.kt:175]: Same here

**Review by zsmb13 — CHANGES_REQUESTED** [review]
Implementation looks all good, let's clean up the Dependencies file a little though 😄

**Review by zsmb13 — APPROVED** [review]
Good to go as far as I'm concerned 👍

**Review by JcMinarro — APPROVED** [review]

**Pull request merged**